conans/client/conf/detect.py

```diff
@@ -244,6 +244,18 @@ def _detect_compiler_version(result, output, profile_path):
             result.append(("compiler.libcxx", "libstdc++"))
     elif compiler == "sun-cc":
         result.append(("compiler.libcxx", "libCstd"))
+    elif compiler == "lcc":
+        result.append(("compiler.base", "gcc"))  # do the same for Intel?
+        result.append(("compiler.base.libcxx", "libstdc++"))
+        version = Version(version)
```

Comment:
don't like this code to be here and to be like that. ideally, `CompilerId` may detect base compiler (by inspecting `__GNUC__` in addition to the `__LCC__`, but we still have old `--version`-based detection.